.env

```
@@ -143,6 +143,6 @@ _APP_WEBHOOK_MAX_FAILED_ATTEMPTS=10
 _APP_PROJECT_REGIONS=default
 _APP_FUNCTIONS_CREATION_ABUSE_LIMIT=5000
 _APP_STATS_USAGE_DUAL_WRITING_DBS=database_db_main
-_APP_TRUSTED_HEADERS=x-forwarded-for
+_APP_TRUSTED_HEADERS=x-forwarded-for,x-forwarded-proto,x-forwarded-host,x-forwarded-port
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The _APP_TRUSTED_HEADERS key should go before the _APP_USAGE_AGGREGATION_INTERVAL key

(UnorderedKey)